lib/editing/editor_codemirror.dart

```diff
@@ -261,7 +261,7 @@ class _CodeMirrorEditor extends Editor {
   bool get readOnly => cm.getReadOnly();
 
   @override
-  set readOnly(bool ro) => cm.setReadOnly(ro, true);
+  set readOnly(bool ro) => cm.setReadOnly(ro, false);
```

Comment:
`false` is the default, so:

```
cm.setReadOnly(ro);
```

Should be enough here.

Reply:
Updated code as expected.